Corona-Warn-App/src/main/res/layout/include_certificate_qrcode_card.xml

```diff
@@ -6,29 +6,40 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginHorizontal="24dp"
-    android:padding="16dp">
+    android:padding="8dp">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="8dp" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="8dp" />
 
     <TextView
         android:id="@+id/covpass_info_title"
         style="@style/body2"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginEnd="@dimen/spacing_small"
+        android:layout_marginTop="8dp"
         android:paddingBottom="@dimen/spacing_small"
         android:text="@string/person_details_certificate_covpass_info_title"
         app:layout_constraintEnd_toStartOf="@id/covpass_info_button"
-        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
         app:layout_constraintTop_toTopOf="parent" />
 
     <ImageButton
         android:id="@+id/covpass_info_button"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
-        android:adjustViewBounds="true"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
```

Comment:
we have dimension constants that you can use instead of hard-coded values eg. `spacing_large = 48dp `, `spacing_tiny = 8dp`

Reply:
I introduced a style that uses non-hardcoded values now.